### PR TITLE
Add @kadira/storybook-addons into resolve.alias.

### PR DESCRIPTION
**Pull request by arunoda**
Then addons will work on NPM2 properly.
Related issue: https://github.com/kadirahq/storybook-addons/issues/8

### CODE REVIEW

File: src/server/config.js [hunk 1]
```diff
@@ -132,5 +132,12 @@ export default function (configType, baseConfig, configDir) {
         ...customConfig.module.loaders || [],
       ],
     },
+    resolve: {
+      ...customConfig.resolve,
+      alias: {
+        ...config.alias,
+        ...customConfig.alias,
```
Comment (stipsan) [inline src/server/config.js]: Shouldn't this be ...customConfig.resolve.alias?

**Comment by arunoda** [comment]
Oh. Yes. Oops.

File: src/server/config.js (hunk 1 above)
Comment (arunoda) [inline src/server/config.js]: Oops. Fixing this.

Comment (arunoda) [inline src/server/config.js]: Did a hotfix. Try now.

Comment (stipsan) [inline src/server/config.js]: Thanks, it's resolved!

**Pull request merged**